superset-frontend/src/views/CRUD/welcome/Welcome.tsx

```diff
@@ -27,6 +27,7 @@ import {
   createErrorHandler,
   getRecentAcitivtyObjs,
   mq,
+  getMineObjs,
```

Comment:
This is a stupid nit, but I'll throw it out there... `Mine` makes this explicit about what it's used for (which I like) but there's something in me that wants to rename this to `getMyObjs` since it just _sounds_ better. We should probably leave it alone.... but just it does sound weird to my brain, so I'm just just hemming and hawing about it :P

Reply:
Make sense. I originally want to go with that naming

Reply:
How about `getUserOwnedObjects`?